packages/babel-preset-typescript/test/fixtures/jsx-compat/js-valid/output.js

```diff
@@ -0,0 +1,6 @@
+var _reactJsxRuntime2 = require("react/jsx-runtime");
```

Comment:
Can you remove this file? It has been removed in https://github.com/babel/babel/pull/11316/files#diff-a4501ac491dd4222ff6be1987b1c8ba6

Reply:
Done.